### PR TITLE
Guide Spring Boot to Micronaut UriComponentsBuilder vs UriBuilder

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

File: guides/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Building URIs - Spring Boot to Micronaut Framework",
+  "intro": "This guide shows compares Spring's UriComponentsBuilder vs a Micronaut Framework's UriBuilder.",
```
Comment (alvarosanchez) [inline guides/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder/metadata.json]:
```suggestion
  "intro": "This guide compares using Spring's UriComponentsBuilder vs Micronaut Framework's UriBuilder.",
```

File: guides/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
+common:header-top.adoc[]
+
+== Sample Project
+
+You can link:@sourceDir@.zip[download a sample application] with the code examples in this article.
+
+== Introduction
+
+While developing web applications, we often need to build URIs. For example, to create redirection responses. In this article, we compare two similar APIs - Spring
+https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/util/UriComponentsBuilder.html[`UriComponentsBuilder`] and Micronaut https://docs.micronaut.io/latest/api/io/micronaut/http/uri/UriBuilder.html[UriBuilder].
```
Comment (alvarosanchez) [inline guides/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder.adoc:10]:
```suggestion
While developing web applications, we often need to build URIs. For example, to create redirection responses. In this article, we compare two similar APIs - Spring's
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/util/UriComponentsBuilder.html[`UriComponentsBuilder`] and Micronaut's https://docs.micronaut.io/latest/api/io/micronaut/http/uri/UriBuilder.html[UriBuilder].
```

**Review by alvarosanchez — APPROVED** [review]

File: guides/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder.adoc (hunk 2 above)
Comment (sdelamo) [reply guides/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder.adoc:10]: We cannot use Micronaut as a noun if we want to follow the brand guidelines

Comment (graemerocher) [reply guides/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder/spring-boot-to-micronaut-uri-components-builder-vs-uri-builder.adoc:10]: but you used it as noun in the original text? All Alvaro added was the possessive

**Pull request merged**